### DISCUSSION TITLE
Why the hammerDB v4.4 on windows 10 always crash ?

### DISCUSSION THREAD

**Discussion by tommysway** [TPC-Council/HammerDB#354] [General]
The test window often disappeared for no apparent reason and the test was terminated.
How can I locate the specific cause of the crash ?

**Comment by tommysway** [comment]
It's Oracle Bug 12733000  , I resolved it.

**Reply by sm-shaw** [reply]
This is in the documentation release notes https://www.hammerdb.com/docs/ch01s01.html#d0e52 as follows:

Oracle on Windows: Oracle Bug 12733000 OCIStmtRelease crashes or hangs if called after freeing the service context handle

If you are running HammerDB against Oracle on Windows there is long established bug in Oracle that can cause application crashes for multi-threaded applications on Windows.This bug can be investigated on the My Oracle Support website with the following reference. Bug 12733000 OCIStmtRelease crashes or hangs if called after freeing the service context handle. To resolve this Oracle issue add the following entry to the SQLNET.ORA file on your HammerDB client.

```
SQLNET.AUTHENTICATION_SERVICES = (NTS)
DIAG_ADR_ENABLED=OFF 
DIAG_SIGHANDLER_ENABLED=FALSE
DIAG_DDE_ENABLED=FALSE
```

**Reply by tommysway** [reply]
Thank you!
